packages/assets-controllers/src/NftDetectionController.ts

```diff
@@ -330,7 +333,23 @@ export type Attributes = {
   createdAt?: string;
 };
 
-export type Collection = {
+export type GetCollectionsResponse = {
+  collections: CollectionResponse[];
+};
+
+export type CollectionResponse = {
```

Comment:
Can you enumerate the specific list of fields this PR is adding to state?  Or do a before/after diff of the state for an NFT?  When exporting my existing state, I think I already see most of these fields.  Is it just creator and ownerCount being added?

Reply:
The fields that are added are the same in the CollectionResponse type:

```
creator
openseaVerificationStatus
contractDeployedAt
ownerCount
tokenCount
topBid
```

After checking the comment below 👇 ;

I think the only field that i might have mistakenly added and can be removed is the tokenCount field being returned by both getUserTokens API and getTokensApi.
The getUserTokens API is the one being used like you know for the NFT detection. Otherwise, we use the getUserTokens API to fetch the NFT information.


| field                                         |    /userTokens      |         /tokens                                                                                                        
|-----------------------------|----------------- |---------------
| creator                                    |             No            |            Yes
| openseaVerificationStatus.   |             Yes          |             No
| contractDployedAt                 |             Yes           |            No
| ownerCount                            |            No             |            No
| tokenCount                             |            Yes            |           Yes
| topBid                                      |               No          |            No



The `topBid` field returned in both APIs is the token Top Bid. The topBid returned in the /collections API is the collection topBid. On the client side we will check the max between both and display it.